vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -386,6 +388,16 @@ void vtkF3DRenderer::ConfigureRenderPasses()
   // Image post processing passes
   vtkSmartPointer<vtkRenderPass> renderingPass = newPass;
 
+  if (this->AntiAliasingModeEnabled == vtkF3DRenderer::AntiAliasingMode::SSAA)
+  {
+    vtkNew<vtkSSAAPass> ssaaP;
+#if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 4, 20250329)
```

Comment:
we need to wait for this to be merged though

Reply:
of course